api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -282,7 +282,7 @@ type VeleroConfig struct {
 	NoDefaultBackupLocation bool `json:"noDefaultBackupLocation,omitempty"`
 	// Pod specific configuration
 	PodConfig *PodConfig `json:"podConfig,omitempty"`
-	// Velero server’s log level (use debug for the most logging, leave unset for velero default)
+	// Velero server's log level (use debug for the most logging, leave unset for velero default)
```

Comment:
curious if this was your change or the ide's linter?

Reply:
IDE made this.